pkg/draft/server.go

```diff
@@ -19,10 +19,10 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"k8s.io/api/core/v1"
 	apiErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	k8s "k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/pkg/api/v1"
```

Comment:
how come we're switching to use kubernetes core here instead of client-go?

Reply:
Oh, I see this was done because of a dependency conflict with Helm?

Reply:
Yeah, it's due to upgrading to helm 2.7, which supports kube 1.8. We'll have to do the same thing when we bump to helm 2.8 (kube 1.9)

Reply:
You can see @adamreese's changes to helm for kube 1.8 support is similar: https://github.com/Azure/draft/blob/5937caa7c2e678c25961b7d5699bea6fc2aca6e4/scripts/setup-apimachinery.sh#L27-L35